### PR TITLE
LPS-71182 - If workflow assigns by Organization Role, and the workflOw is applied to a regular site, user with organization role should not be notified

### PR DESCRIPTION
**Pull request by daviddotzhang**

### CODE REVIEW

**Comment by natocesarrego** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/display/context/WorkflowTaskDisplayContext.java [hunk 1]
```diff
@@ -1037,14 +1039,42 @@ protected WorkflowTaskSearch searchTasks(boolean searchByUserRoles)
...
 		return searchContainer;
 	}
 
+	protected void setAuthorizedSearchContainerResults(List<WorkflowTask> 
+		workflowTasks, WorkflowTaskSearch searchContainer){
+
+		ThemeDisplay themeDisplay =
+			_workflowTaskRequestHelper.getThemeDisplay();
+
+		List<WorkflowTask> authorizedWorkflowTasks = new ArrayList();
```
Comment (natocesarrego) [inline modules/apps/forms-and-workflow/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/display/context/WorkflowTaskDisplayContext.java]: @daviddotzhang, there is any reason for you don't include the ArrayList type during the instantiation? You can also use the diamond operator to do it, like ArrayList<>();

File: modules/apps/forms-and-workflow/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/display/context/WorkflowTaskDisplayContext.java [hunk 2]
```diff
@@ -1037,14 +1039,42 @@ protected WorkflowTaskSearch searchTasks(boolean searchByUserRoles)
...
+
+		for (int i = 0; i < workflowTasks.size(); i++) {
```
Comment (natocesarrego) [inline modules/apps/forms-and-workflow/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/display/context/WorkflowTaskDisplayContext.java]: @daviddotzhang, there is any special reason for you use a regular **for** instead a **enhanced for**?

**Comment by daviddotzhang** [comment]
@natocesarrego I updated the code, and sent to you here.
https://github.com/natocesarrego/liferay-portal/pull/78

**Pull request closed without merging**